<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="LP002572" failures="0" tests="2" name="com.coretex.tests.i360.BestWesternWF1" time="262.432" errors="1" timestamp="2022-06-27T17:58:42 CDT" skipped="0">
  <testcase classname="com.coretex.tests.i360.BestWesternWF1" name="TC_BestWesternWF2" time="160.769">
    <error message="Unable to validate the summary page 
Element Reservation Summary Page is not Displayed on UI" type="java.lang.Exception">
      <![CDATA[java.lang.Exception: Unable to validate the summary page 
Element Reservation Summary Page is not Displayed on UI
at com.coretex.pageobjects.i360.SelectRoomPage.verifySummaryPage1(SelectRoomPage.java:99)
at com.coretex.tests.i360.BestWesternWF1.TC_BestWesternWF2(BestWesternWF1.java:47)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
    </error>
  </testcase> <!-- TC_BestWesternWF2 -->
  <system-out/>
  <testcase classname="com.coretex.tests.i360.BestWesternWF1" name="TC_BestWesternWF1" time="101.663"/>
  <system-out/>
</testsuite> <!-- com.coretex.tests.i360.BestWesternWF1 -->
